ColumbusStateWorkforceInnovation/java-3-model-view-controller-lab
# Correct Project Setup

## Context
Students are reporting that they are unable to import this project into Intellij after cloning it down. This PR rebuilds the project structure by renaming the .iml file.

### Changes
- Adding project files back in.
- Adding JUnit back in.

### Validation
1. Clone this project.
2. Pull this feature branch.
3. Open the project in Intellij.
4. Verify that the project loads correctly.
5. Verify that the tests can be run correctly.
